tests/instances.spec.ts

```diff
@@ -188,31 +168,45 @@ test("instance edit cloud init configuration", async ({ page }) => {
   await assertCode(page, "Network config", "foo:");
   await assertCode(page, "User data", "bar:");
   await assertCode(page, "Vendor data", "baz:");
+});
 
+test("instance terminal operations", async () => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
```

Comment:
Not reusing our browser-name-based `instance` here?

Reply:
running terminal operations on an existing instance created outside of this test consistently fails for some reason. I think it may be related to configurations set in another test that comes before this one but couldn't nail it down to what it was. So I left the instance creation inside of this test instead.

Reply:
> I think it may be related to configurations set in another test that comes before this one

If that's the case, it should work if this test is done before any other? Have you tried that?

Reply:
I have run the test by itself and it passed, but I didn't try to move it before other tests and run it like that, I will try that quick.

Reply:
Okay moving it to the beginning of instance tests allows it to pass, but in addition the instance needs to be stopped for the subsequent tests to pass so I've added that as well.